### PR TITLE
feat(paragraph): Add Paragraph component

### PR DESCRIPTION
**Pull request by pepeladeira**
A Paragraph component (`<P />`) for write content.

Also apply it on `VercelInviteUser` preview.

### CODE REVIEW

File: packages/p/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "scripts": {
+    "build": "tsup src/index.ts --format esm,cjs --dts --external react",
+    "dev": "tsup src/index.ts --format esm,cjs --dts --external react --watch",
+    "lint": "TIMING=1 eslint src/**/*.ts* --fix",
+    "clean": "rm -rf dist",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "react": "^18.2.0"
```
Comment (bukinoshita) [inline packages/p/package.json:22]: Can you pin the react version?

File: packages/p/src/p.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+type RootProps = React.ComponentPropsWithoutRef<'p'>;
+
+export interface PProps extends RootProps {}
+
+export const P = React.forwardRef<PElement, Readonly<PProps>>(
+  ({ style, ...props }, forwardedRef) => (
+    <p
+      ref={forwardedRef}
+      style={{
+        color: "#000",
```
Comment (bukinoshita) [inline packages/p/src/p.tsx:13]: I think the default should be inherited from parent.

```suggestion
```

**Comment by bukinoshita** [comment]
Can we also rebase with https://github.com/zenorocha/react-email/pull/47

File: packages/p/package.json (hunk 1 above)
Comment (pepeladeira) [reply packages/p/package.json:22]: okay

File: packages/p/src/p.tsx (hunk 2 above)
Comment (pepeladeira) [reply packages/p/src/p.tsx:13]: great suggestion

File: packages/p/src/p.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        fontFamily: "-apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif",
+        fontSize: "14px",
+        lineHeight: "24px",
```
Comment (zenorocha) [inline packages/p/src/p.tsx:16]: Add `margin: "16px 0px"` to make sure it's standardized across different email clients.

**Review by zenorocha — CHANGES_REQUESTED** [review]

**Comment by bukinoshita** [comment]
We also may want to add the component in the root readme component list

File: packages/p/package.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@react-email/p",
+  "version": "0.0.1",
+  "description": "A Paragraph component for write your content",
```
Comment (zenorocha) [inline packages/p/package.json:4]: `A paragraph is a block of text separated by some white space`

**Review by zenorocha — CHANGES_REQUESTED** [review]
Can you also add this and the Hr link to the root's readme? https://github.com/zenorocha/react-email#components

File: packages/p/src/p.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+export interface PProps extends RootProps {}
+
+export const P = React.forwardRef<PElement, Readonly<PProps>>(
+  ({ style, ...props }, forwardedRef) => (
+    <p
+      ref={forwardedRef}
+      style={{
+        color: "#000",
+        fontFamily: "-apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif",
```
[the remark this answers is not in the document]
Comment (pepeladeira) [reply packages/p/src/p.tsx]: understood!

File: packages/p/package.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@react-email/p",
+  "version": "0.0.1",
+  "description": "A paragraph is a block of text separated by some white space",
```
Comment (bukinoshita) [inline packages/p/package.json:4]: Maybe we can omit the "A paragraph is.."

```suggestion
  "description": "A block of text separated by some white space",
```

Comment (pepeladeira) [reply packages/p/package.json:4]: good point

Comment (pepeladeira) [reply packages/p/package.json]: done!

**Pull request merged**